### DISCUSSION TITLE
FSEQ Debug Output

### DISCUSSION THREAD

**Discussion by DrAnonymous** [forkineye/ESPixelStick#435] [Q&A]
I had an issue getting an fseq file to play from XLights. I figured out that the format needed to be changed to uncompressed for it to work when copied to the SD card.

Is there any diagnostic log that outputs from the v4 FW? It would have been really helpful to see terminal input with something like:

- Opening file: <file name>.fseq
- No valid data found
- File closed

**Comment by forkineye** [comment]
Currently, debugging output only logs to the serial console.  If you need access to the USB port while using an ESPixelStick V3, you can use stackable wemos shield headers to raise the module and make room for the connection.
